packages/shared/lib/core/index.ts

```diff
@@ -1 +1 @@
-export default {}
+export * from './router/'
```

Comment:
It might be better to name the export here once other exports are added, e.g. `export * as router from './router`. It should work anyways with the proper barrel file setup, but I have a weird preference to explicitly name it :sweat_smile:.